setup/security/ssl.md

```diff
@@ -313,27 +367,42 @@ If you have problems...
 * If you are running Spinnaker in a distributed environment, have you run
 `hal deploy connect`?
 
-## About network configurations
-
-Each authentication mechanism is configured differently depending on where the
-SSL connection terminates.
-
-### Server-terminated SSL
 
-Terminating SSL within the Gate server is the de-facto way to enable SSL for
-Spinnaker. This works with or without a load balancer proxying traffic to this
-instance.
+## Using a custom CA for internal communications
+There are a lot of places in Spinnaker which support the ability to configure custom Java trust/key stores for 
+organizations who use internally signed certificates. In some cases, however, this isn’t supported yet but you still 
+need to talk to a service which serves one of these certificates. This post will show you how to import your 
```

Comment:
"This section" maybe? "This post" sounds like a blog post.

Reply:
It was a blog post ;) . really a KB article.  But it's relevant and so copied it over